config/locales/en/loans.en.yml

```diff
@@ -12,6 +12,8 @@ en:
     confirm_response_deletion: "WARNING: Are you sure you want to permanently delete these responses?"
     criteria: "Criteria"
     criteria_not_entered: "Criteria have not been entered yet."
+    default_name_message: "If left blank, a name will be automatically generated for this loan."
+    default_loan_name: "%{org} - %{date}"
```

Comment:
Removes the need to use `.html_safe` in the methods and as a bonus adds internationalization. I did replace `&ndash;` with `-`, which should be fine.